fluent-package/yum/fluent-package.spec.in

```diff
@@ -165,22 +165,21 @@ if [ $1 -eq 1 ]; then
     fi
 fi
 if [ -d "%{_sysconfdir}/prelink.conf.d/" ]; then
-  echo "prelink detected. Installing /etc/prelink.conf.d/td-agent-ruby.conf ..."
-  if [ ! -f %{_sysconfdir}/prelink.conf.d/td-agent-ruby.conf ]; then
-      %{__cp} -f /opt/td-agent/share/td-agent-ruby.conf %{_sysconfdir}/prelink.conf.d/td-agent-ruby.conf
-  else
-    if [ $(grep '\-b /opt/td-agent/embedded/bin/ruby' -c %{_sysconfdir}/prelink.conf.d/td-agent-ruby.conf) -eq 1 ]; then
-      echo "old-prelink detected, Update /etc/prelink.conf ..."
-      %{__sed} -i"" %{_sysconfdir}/prelink.conf.d/td-agent-ruby.conf -e "s,/embedded/,/,"
-    fi
+  # Drop prelink itself which is used until v4
+  echo "prelink detected. checking /etc/prelink.conf.d/ ..."
+  if [ -f %{_sysconfdir}/prelink.conf.d/td-agent-ruby.conf ]; then
+    echo "Removing prelink settings for td-agent from /etc/prelink.conf ..."
```

Comment:
The message doesn't seem match with actual behaviour.

Reply:
Fixed.